planning/behavior_path_planner/include/behavior_path_planner/utils/avoidance/helper.hpp

```diff
@@ -190,22 +198,34 @@ class AvoidanceHelper
     return *itr;
   }
 
-  boost::optional<double> getFeasibleDecelDistance(const double target_velocity) const
+  double getFeasibleDecelDistance(const double target_velocity) const
```

Comment:
I thouht this function can be consolidated with next function intuitively.
But you have might considered this option and intentionally divide those function correct?

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3978/commits/23940ea4f8b93d6864a6352b29286e624479b473.